web/src/components/__tests__/ListItem.spec.tsx

```diff
@@ -1,18 +1,22 @@
-import React from 'react'
-import { shallow } from 'enzyme'
+import React, { FunctionComponent } from 'react'
+import { render } from '@testing-library/react'
 
 import ListItem from '../ListItem'
+import wrapWithTheme from '../../testing/wrapWithTheme'
 
-describe('ListItem', () => {
-  const path = '/augsburg/en/events/first_event'
+describe('ListItemSpec', () => {
+  const path = 'https://tuerantuer.org'
 
-  it('should render a Link and match snapshot', () => {
-    expect(
-      shallow(<ListItem thumbnail='thumbnail' title='first Event' path={path} isExternalUrl={false} />)
-    ).toMatchSnapshot()
+  it('should not render thumbnail in the ListItem', () => {
+    const { getByRole } = render(<ListItem title='first Event' path={path} />, {
+      wrapper: wrapWithTheme as FunctionComponent
```

Comment:
Why do we need the typecast? Do we need to do this everytime we use `wrapWithTheme`?

Reply:
@f1sh1918 I would extract this in a separate PR and apply this to where neccessary/possible